src/main/java/model/OperationType.java

```diff
@@ -0,0 +1,9 @@
+package model;
+
+//ENUM for operation types.
+public enum OperationType {
+    B,
```

Comment:
do not use abbreviated names
```suggestion
    BALANCE("b"),
```

Reply:
OK. But it's not clear to me how the constructor in ENUM works?